### ISSUE TITLE
What is hdf5

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/hdf5#1]
https://www.neonscience.org/resources/learning-hub/tutorials/about-hdf5

Example of file from: https://ic.bbp.epfl.ch/icproject

[rCell23970.nwb.zip](https://github.com/cheminfo/hdf5/files/10113357/rCell23970.nwb.zip)

**Comment by targos** [comment]
> Example of file from: https://ic.bbp.epfl.ch/icproject

This link is not public.

**Comment by targos** [comment]
HDF5 is very similar to NetCDF, so maybe we should keep only one parser for less maintenance cost.

**Comment by lpatiny** [comment]
Yes indeed we need to check if it can be added in the netCDF parser.

There is a java code that read this file format https://www.hdfgroup.org/downloads/hdfview/

On the website https://ic.bbp.epfl.ch/icproject they say it is public ;)

![image](https://user-images.githubusercontent.com/1484241/204543567-8912c1c1-63a8-441f-a728-60fd266d7220.png)

**Comment by ghost** [comment]
Interesting, I've seen it in python for ML, I think to store images. I'll read about it.

**Comment by ghost** [comment]
I've read part of [the specification manual here](https://docs.hdfgroup.org/hdf5/develop/_f_m_t3.html#ThisDocument), and wrote just some code to test, it seems possible to write it. We may shortcut / save time using the source code of the viewer, from the link you send previously as well.  

But is there any reason why [jsfive](https://github.com/usnistgov/jsfive) isn't useful? The license says it is in the public domain. Looking in the code is based off ([pyfive](https://github.com/jjhelmus/pyfive)) is BSD3.0 which still can be embedded anywhere I think.

Let me know just to know whether I should keep reading it or not.

**Comment by lpatiny** [comment]
Example of file

[Data_2023.05.19_15h01m27s.h5.zip](https://github.com/cheminfo/hdf5/files/11869949/Data_2023.05.19_15h01m27s.h5.zip)
